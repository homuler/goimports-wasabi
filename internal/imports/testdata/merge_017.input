// merge empty import declarations
package main

import () // postrparen comment (1)
import () // postrparen comment (2)
import () // postrparen comment (3)
