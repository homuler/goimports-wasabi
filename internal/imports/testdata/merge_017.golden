// merge empty import declarations
package main

import () // postrparen comment (1)
// postrparen comment (2)
// postrparen comment (3)
